test/Microsoft.AspNetCore.Mvc.Core.Test/Infrastructure/ActionContextAccessorTests.cs

```diff
@@ -16,7 +16,7 @@ private static void DomainFunc()
             accessor.ActionContext = new ActionContext();
         }
 
-        [Fact]
+        [Fact(Skip = "https://github.com/Microsoft/vstest/issues/419")]
```

Comment:
Try the AppDomainSetup.ApplicationBase workaround I mentioned in microsoft/vstest#419